src/applications/letters/containers/AddressSection.jsx

```diff
@@ -55,9 +55,7 @@ export function AddressSection({ address }) {
 
   return (
     <>
-      <div aria-live="polite" aria-relevant="additions">
```

Comment:
These live regions weren't adding context for screen readers and were creating a lot of noise when I expanded the download letter accordions on the next page. Removing them lets users focus on the cues like buttons being expanded. This update makes the local Benefit Letters match our default `<VaAccordion>` experience.